hack/dev_devpod.sh

```diff
@@ -0,0 +1,13 @@
+#! /usr/bin/env zsh
+
+set -e
+
+NS=${1:-"default"}
+
+if [[ ! $PWD == *"/go/src/devpod"* ]]; then
+  echo "Please run this script from the /workspace/loft/devpod directory"
+  exit 1
+fi
+
+CGO_ENABLED=1 go build -ldflags "-s -w" -tags profile -race -o devpod-cli
```

Comment:
Which library needs Glibc? 🤔 
`07:24:02 info /usr/local/bin/devpod: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /usr/local/bin/devpod)`

Reply:
CGO, it's used for the race detector. I need to update the script to make this optional

Reply:
Oh I wouldn't have expected the race detector to require CGO but alas
https://github.com/golang/go/issues/9918
https://github.com/golang/go/issues/6508